.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
+.vscode
```

Comment:
(not a big deal, but this seems like the sort of thing to put in your global gitconfig, rather than adding specific editor knowledge to individual repos)

Reply:
(reminder)